SilKit/source/core/vasio/VAsioDatatypes.hpp

```diff
@@ -147,6 +147,13 @@ struct ProxyMessage
     std::vector<uint8_t> payload;
 };
 
+enum class MessageAggregationKind : uint8_t
+{
+    UserDataMessage = 0,
+    TimeSyncMessage = 1,
```

Comment:
I think we need more types that trigger the flush, so we should call this `FlushAggregationMessage` or similar

Reply:
Done.